product_profile/models/product_profile.py

```diff
@@ -25,12 +25,11 @@ def format_except_message(error, field, self):
     model = type(self)._name
     message = _(
         "Issue\n------\n"
-        "%s\n'%s' value can't be applied to '%s' field."
+        f"{error}\n {value} value can't be applied to {field} field."
         "\nThere is no matching value between 'Product Profiles' "
-        "\nand '%s' models for this field.\n\n"
+        f"\nand {model} models for this field.\n\n"
```

Comment:
You cannot use f-strings in translation methods as it will be formatted before translation evaluation.

Use either `_(...%(model)s...) % dict(model=model)`or `_(...{model}...).format(model=model)`